packages/compass-import-export/src/modules/export.js

```diff
@@ -126,12 +128,18 @@ const reducer = (state = INITIAL_STATE, action) => {
   }
 
   if (action.type === ON_MODAL_OPEN) {
-    return {
+    const newState = {
       ...INITIAL_STATE,
       count: action.count,
       query: action.query,
+      aggregation: action.aggregation || {},
```

Comment:
Empty object is a pretty inconvenient value to represent "empty" that is hard to check in the other code, I'd suggest to have it set to `null` instead (similar for the query). Also as you only can have one or another, maybe it's worth having some sort of "type" value in the state